source/common/network/io_socket_handle_impl.h

```diff
@@ -94,6 +95,8 @@ class IoSocketHandleImpl : public IoSocketHandleBaseImpl {
                            CMSG_SPACE(sizeof(struct in6_pktinfo)) + CMSG_SPACE(sizeof(uint16_t))};
 
   const bool udp_read_normalize_addresses_;
+
+  const bool receive_ecn_;
```

Comment:
nit: brief comment that this latches a copy of the runtime feature "envoy.reloadable_features.quic_receive_ecn".